pkgs/applications/networking/instant-messengers/beeper/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, mkDerivation, fetchurl, appimageTools, imagemagick, libsecret, ... }:
+let
+  pname = "beeper";
+  version = "3.64.5";
+  src = fetchurl {
+    url = "https://download.beeper.com/linux/appImage/x64";
```

Comment:
This url is not stable. It always is the latest public release. It will result in nondeterministic builds.